lib/determinator/tracking/tracker.rb

```diff
@@ -9,7 +9,8 @@ class Tracker
       def initialize(type)
         @determinations = []
         @type = type
-        @start = now
+        @monotonic_start = now
+        @start = Time.now
```

Comment:
Why do u measure the time in two different ways?

Reply:
It's explained in the description of the PR ;) `Process.clock_gettime(Process::CLOCK_MONOTONIC)` is platform-specific and can't be easily converted to a timestamp (it can be the time from reboot for example).